packages/ListBox/src/components/Option/Option.styles.js

```diff
@@ -30,13 +31,13 @@ export const OptionStyled = styled.li`
 
     const isActived = props.isActive
       ? `
-        border: 3px solid Highlight;
+        border: 2px solid Highlight;
         border-radius: 3px;
         `
       : "";
 
     const hasPreventDefaultOnSelect =
-      props.isActive && props.hasPreventDefaultOnSelect ? `border: 2px solid ${tokens.color.blackLighten60}` : "";
+      props.isActive && props.hasPreventDefaultOnSelect ? `border-color: ${tokens.color.blackLighten60}` : "";
 
     const isSelected = props.isSelected ? `font-weight: 600; background: ${blueSelected}` : "";
```

Comment:
While we're here, can we remove the `font-weight: 600`? I believe I argued successfully with @sunminsarahlee that we shouldn't change font weights for this kind of "fleeting" state change.